modules/sdk/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplateFilesTest.java

```diff
@@ -446,8 +446,8 @@ private void _testLiferayPluginPackageProperties(Path path)
 		_testPropertyValue(path, properties, "author", "${author}");
 		_testPropertyValue(path, properties, "change-log", "");
 		_testPropertyValue(path, properties, "licenses", "LGPL");
-		_testPropertyValue(
-			path, properties, "liferay-versions", "7.0.0+,7.1.0+");
+		_testPropertyContainsValue(
```

Comment:
Hey @achaparro, 

We'll need to leave the check for 7.0.0+ in there because when we build Blade CLI it still uses the latest version of project templates for our customers on 7.0